### ISSUE TITLE
AttributeError: __exit__ and __enter__ are missing in BlankSpan class. 

### ISSUE THREAD

**Issue by mayurkale22** [census-instrumentation/opencensus-python#312]
The magic methods ```__exit__``` and ```__enter__``` are missing in BlankSpan class. 
```
from opencensus.trace import Span

with Span(name="foo") as s:
    print(s.name)

foo
```
BUT

```
from opencensus.trace.blank_span import BlankSpan 

with BlankSpan(name="foo") as s:
    print(s.name)

AttributeError: __exit__
```

One way to guarantee that both Span and BlankSpan are compatible is to create a common interface that they both implement. This is how it is done for tracers for instance.

**Comment by mayurkale22** [comment]
@tgermain Thanks for informing us.

**Comment by mayurkale22** [comment]
Closed in #313
